src/ring_theory/power_basis.lean

```diff
@@ -143,11 +143,7 @@ open_locale big_operators
 
 variable [algebra A S]
 
-/-- `pb.minpoly_gen` is a minimal polynomial for `pb.gen`.
-
-If `A` is not a field, it might not necessarily be *the* minimal polynomial,
```

Comment:
why did we get rid of this?

Reply:
I didn't remove the docstring, just changed "a" to "the" and removed the second paragraph which is now obsolete.

Reply:
Ah, cool, I didn't realise it was obsolete now.